game/data/groups.py

```diff
@@ -0,0 +1,88 @@
+from enum import Enum
+
+
+class GroupRole(Enum):
+    Unknow = "Unknown"
+    AntiAir = "AntiAir"
+    Building = "Building"
+    Naval = "Naval"
+    GroundForce = "GroundForce"
+    Defenses = "Defenses"
+    Air = "Air"
```

Comment:
I added "Air" because i've made the `UnitType` an abstract base class which has a "UnitClass" attribute and AirCrafttype inherits from this UnitType. So therefore all Aircraft got the `Air` UnitClass. from an earlier version of the implementation i had the same implementation for task and Role but i removed it before i openend the PR. So this is a remnant as well. :D Will be removed.